### PR TITLE
Goth test: provider resumes work after requestor abandons a task

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**
- new WASI task package: sleep for N seconds

### CODE REVIEW

**Comment by mfranciszkiewicz** [comment]
blocked by #1527

File: goth_tests/domain/ya-provider/test_provider_multi_activity.py [hunk 1]
```diff
@@ -159,3 +163,106 @@ async def test_provider_single_simultaneous_activity(
...
+                )
+                .property("golem.srv.caps.multi-activity", True)
+                .property("golem.com.payment.debit-notes.accept-timeout?", 5)
+                .constraints(
+                    "(&(golem.com.pricing.model=linear)\
+                    (golem.srv.caps.multi-activity=true)\
+                    (golem.runtime.name=wasmtime))"
+                )
+                .build()
+            )
```
Comment (nieznanysprawiciel) [inline goth_tests/domain/ya-provider/test_provider_multi_activity.py:212]: What is the benefit of defining function inside other function over defining it on top level?

File: goth_tests/domain/ya-provider/test_provider_multi_activity.py [hunk 2]
```diff
@@ -159,3 +163,106 @@ async def test_provider_single_simultaneous_activity(
...
+
+            exe_script = wasi_exe_script(runner)
+            activity_id = await requestor.create_activity(agreement_id)
+            await provider.wait_for_exeunit_started()
+
+            batch_id = await requestor.call_exec(activity_id, json.dumps(exe_script))
+            await requestor.collect_results(
+                activity_id, batch_id, len(exe_script), timeout=30
+            )
+            await requestor.destroy_activity(activity_id)
```
Comment (nieznanysprawiciel) [inline goth_tests/domain/ya-provider/test_provider_multi_activity.py:227]: Most of this function can be replaced with `goth_tests.helpers.activity.run_activity`

**Review by nieznanysprawiciel — APPROVED** [review]

File: goth_tests/domain/ya-provider/test_provider_multi_activity.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply goth_tests/domain/ya-provider/test_provider_multi_activity.py:212]: It's next to other test-specific functions. Other tests wouldn't benefit from having it at top level

File: goth_tests/domain/ya-provider/test_provider_multi_activity.py (hunk 2 above)
Comment (mfranciszkiewicz) [reply goth_tests/domain/ya-provider/test_provider_multi_activity.py:227]: That's true, but this test doesn't care about `wait_for_exeunit_finished` which assumes computation success.

**Pull request merged**